xarray/testing/assertions.py

```diff
@@ -162,13 +179,13 @@ def assert_identical(a: DataTree, b: DataTree, from_root: bool = True): ...
 
 
 @ensure_warnings
-def assert_identical(a, b, from_root=True):
+def assert_identical(a, b, from_root=True, check_dims="strict"):
```

Comment:
FWIW I would vote to have `assert_identical` be always literally identical, including dim order. But no strong view.